documentation/content/pages/technical-log/December_2023.rst

```diff
@@ -0,0 +1,197 @@
+.. -*- Mode: rst -*-
```

Comment:
Nice doc!

Reply:
Thanks. Will try to keep it updated. Schematic are exported from my Boox e-ink tablet. My dream would be to port XFig on it...